numpy/polynomial/_polybase.py

```diff
@@ -37,6 +37,11 @@ class ABCPolyBase(abc.ABC):
     window : (2,) array_like, optional
         Window, see domain for its use. The default value is the
         derived class window.
+    symbol : str, optional
+        Symbol used to represent the independent variable in string 
+        representations of the polynomial expression, e.g. for printing.
+        The symbol must be a non-empty string where the first character is
+        non-numeric. Default value is 'x'.
```

Comment:
This should be updated to say that the symbol must be a valid Python identifier.